app/main.py

```diff
@@ -1,7 +1,34 @@
+import os
+
+import dotenv
+import requests
+
+
+CITY = "Paris"
+URL = "http://api.weatherapi.com/v1/current.json"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    key = os.getenv("API_KEY")
+    if not key:
+        print("No API key provided")
+        return
+    request = requests.get(
+        URL,
+        params={"key": key, "q": CITY},
+    )
```

Comment:
Consider adding exception handling around the `requests.get` call to manage potential network errors or exceptions.